src/main/java/com/epam/controller/JsonController.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.controller;
+
+import com.epam.model.Passport;
+import com.epam.model.User;
+import com.epam.service.PassportService;
+import com.epam.service.UserService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.transaction.Transactional;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Controller
+@RequestMapping(path = "/json")
+public class JsonController {
+    private UserService userService;
+    private PassportService passportService;
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    public JsonController(UserService userService, PassportService passportService, MappingJackson2HttpMessageConverter converter) {
```

Comment:
do refactoring a string is more 120 signs

Reply:
Done.